shard/src/main/java/com/redhat/developer/shard/ManagerSyncServiceImpl.java

```diff
@@ -0,0 +1,84 @@
+package com.redhat.developer.shard;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.developer.infra.dto.ConnectorDTO;
+import com.redhat.developer.infra.dto.ConnectorStatusDTO;
+import com.redhat.developer.shard.utils.WebClientUtils;
+
+import io.quarkus.scheduler.Scheduled;
+import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.core.Vertx;
+import io.vertx.mutiny.core.buffer.Buffer;
+import io.vertx.mutiny.ext.web.client.HttpResponse;
+import io.vertx.mutiny.ext.web.client.WebClient;
+
+@ApplicationScoped
+public class ManagerSyncServiceImpl implements ManagerSyncService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ManagerSyncServiceImpl.class);
+
+    @ConfigProperty(name = "event-connect.manager.url")
+    String eventConnectManagerUrl;
+
+    @Inject
+    Vertx vertx;
+
+    @Inject
+    OperatorService operatorService;
+
+    WebClient webClient;
+
+    @PostConstruct
+    void init() {
+        webClient = WebClientUtils.getClient(vertx, eventConnectManagerUrl);
```

Comment:
I think it makes more sense to inject the `WebClient` from the IoC context via a `Producer` rather than doing an `@PostConstruct` here.